scripts/vim-patch.sh

```diff
@@ -108,22 +111,23 @@ assign_commit_details() {
     vim_tag="v${1}"
     vim_commit=$(cd "${VIM_SOURCE_DIR}" \
       && git log -1 --format="%H" "${vim_tag}")
-    local strip_commit_line=true
+    local munge_commit_line=true
   else
     # Interpret parameter as commit hash.
     vim_version="${1:0:12}"
+    vim_tag=
     vim_commit=$(cd "${VIM_SOURCE_DIR}" \
       && git log -1 --format="%H" "${vim_version}")
-    local strip_commit_line=false
+    local munge_commit_line=false
   fi
 
   vim_commit_url="https://github.com/vim/vim/commit/${vim_commit}"
   vim_message="$(cd "${VIM_SOURCE_DIR}" \
     && git log -1 --pretty='format:%B' "${vim_commit}" \
       | sed -e 's/\(#[0-9]*\)/vim\/vim\1/g')"
-  if [[ ${strip_commit_line} == "true" ]]; then
+  if [[ ${munge_commit_line} == "true" ]]; then
     # Remove first line of commit message.
-    vim_message="$(echo "${vim_message}" | sed -e '1d')"
+    vim_message="$(echo "${vim_message}" | sed -e '1s/^patch /vim-patch:/')"
```

Comment:
Is this dependent on Bram never changing the message format? In particular, we never want whitespace after `vim-patch:` token.

Reply:
He has a script that handles this.  It [looks like](https://gist.github.com/jamessan/0c8e3770f83c7a3d9eaa204e55eb4245) we're good.  The last 1300 patches or so don't violate that format.